openapi.yaml

```diff
@@ -7457,6 +7511,84 @@ components:
       maxLength: 40
       minLength: 7
       type: string
+    DeployedComponent:
+      additionalProperties: false
+      description: Definition of the deployed software unit.
+      example:
+        repository: github.com/athenianco/athenian-api
+        reference: v0.10.34
+      properties:
+        repository:
+          $ref: '#/components/schemas/RepositoryID'
+        reference:
+          description: |
+            We accept three ways to define a Git reference:
+            1. Tag name.
+            2. Full commit hash (40 characters).
+            3. Short commit hash (7 characters).
+
+            We ignore the reference while we cannot find it in our database. There can be
+            two reasons:
+            - There is a mistake or a typo in the provided data.
+            - We are unable to eventually synchronize with GitHub.
```

Comment:
I'd avoid making the users worry too much about that. Thus say something more like:
"We are temporarily unable to synchronize with GitHub. We'll try again in a moment."